src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/event/PluginPatchsetCreatedEventTest.java

```diff
@@ -85,4 +86,48 @@ public void shouldNotFireOnNoCodeChangeWhenExcluded() {
         patchsetCreated.getPatchSet().setKind(GerritChangeKind.NO_CODE_CHANGE);
         assertFalse(pluginPatchsetCreatedEvent.shouldTriggerOn(patchsetCreated));
     }
+
+    /**
+     * Test that it should, or should not, fire if the commit message matches a regular expression.
+     */
+    @Test
+    public void shouldNotFireIfCommitMessageRegExIsNull() {
```

Comment:
I am in favour of splitting this into multiple tests, however I followed the same "structure" as other tests, where they perform multiple checks.

We should either rename this method, or split into multiple tests.  Thoughts?

Reply:
updated test name